charts/sck-otel/templates/_config-Agent.tpl

```diff
@@ -531,4 +544,19 @@ service:
         - splunk_hec/platformMetrics
         {{- end }}
     {{- end }}
+    {{- if .Values.journaldLogs.enabled }}
```

Comment:
lets also add one more condition 
`splunk-otel-collector.collectLog` to be true. it ensures at least one of endpoint is available for logs